src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements GetFigureArea {
+    private double firstLeg;
+    private double secondLeg;
+
+    @Override
+    public String toString() {
+        return "Figure - RightTriangle: firstLeg " + this.firstLeg
+                + " ,secondLeg " + this.secondLeg
+                + " ,area " + this.getArea() + " color "
+                + this.color;

```

Comment:
The `color` field is accessed directly here. Consider using a getter method if encapsulation is a concern, especially if the `color` field is private in the `Figure` class.